CMakeLists.txt

```diff
@@ -84,6 +84,11 @@ if( "${BACKEND}" STREQUAL "HIP" OR "${BACKEND}" STREQUAL "hip")
     find_package(HIP QUIET)    # Find package HIP
     if(HIP_FOUND)
 
+        # Set external flag to notify GPU_SUPPORT and HIP backend availability - Compatible with rocAL and MIVisionX
+        add_definitions(-DGPU_SUPPORT)
+        set(RPP_BACKEND_HIP 1) # For cmakedefine01 in rpp_backend.h.in
```

Comment:
don't you need to set RPP_BACKEND_OPENCL 0 here

Reply:
I tested it out and #cmakedefine01 seems to define RPP_BACKEND_OPENCL automatically to 0 and the installed include/rpp_backend.h file under /opt/rocm/rpp/include looks like this for the HIP install:
```
#ifndef GUARD_RPP_BACKEND_H_IN
#define GUARD_RPP_BACKEND_H_IN

#define RPP_BACKEND_HIP 1
#define RPP_BACKEND_OPENCL 0

#endif
```